modules/gdscript/gdscript_tokenizer.cpp

```diff
@@ -937,8 +937,10 @@ void GDScriptTokenizerText::_advance() {
 								return;
 							}
 							hexa_found = true;
+						} else if (hexa_found && _is_hex(GETCHAR(i))) {
+
 						} else if (GETCHAR(i) == 'b') {
```

Comment:
What about doing it the same way as the next if? (or alternatively, removing `!hexa_found` from there as well)
```suggestion
						} else if (!hexa_found && GETCHAR(i) == 'b') {
```

Reply:
Yeah. It's unnecessary right now but better to be explicit in case the order gets changed accidentally.